reV/handlers/transmission.py

```diff
@@ -279,23 +279,31 @@ def _calc_cost(distance, line_cost=2279, tie_in_cost=0,
 
         return cost
 
-    def _substation_capacity(self, line_gids):
+    def _substation_capacity(self, gid, line_gids):
         """
         Get capacity of a substation from its tranmission lines
 
         Parameters
         ----------
+        gid : int
+            Substation gid
         line_gids : list
             List of transmission line gids connected to the substation
 
-
         Returns
         -------
         avail_cap : float
             Substation available capacity
         """
 
-        line_caps = [self[gid]['avail_cap'] for gid in line_gids]
+        try:
```

Comment:
Nice, surprised this didn't bite us earlier...

Reply:
it's one of those "fine when it works" type things haha